internal/database/actions.go

```diff
@@ -38,6 +42,60 @@ func ListMeasurements(sess sqlbuilder.Database, resultID int64) ([]MeasurementUR
 	return measurements, nil
 }
 
+type MeasurementWithInput struct {
+	Input string `json:"input"`
+}
+
+func GetMeasurementJSON(sess sqlbuilder.Database, measurementID int64) (map[string]interface{}, error) {
+	var (
+		measurement MeasurementURLNetwork
+		msmtJSON map[string]interface{}
+	)
+
+	req := sess.Select(
+		db.Raw("urls.*"),
+		db.Raw("measurements.*"),
```

Comment:
What does this mean? Select any field from these two tables?

Reply:
Select ~any~**all** fields from these two tables

Reply:
Thanks, I was in doubt!